modules/test/playwright/tests/portal-workflow-task-web/assignments.spec.ts

```diff
@@ -152,11 +144,14 @@ test('send user back to my workflow tasks page after assign another user to revi
 
 	await workflowReviewTasksPage.assignToMenuItem.click();
 
-	await workflowReviewTasksPage.selectAssignee(user2.id.toString());
+	await page.waitForLoadState('networkidle');
 
-	await workflowReviewTasksPage.clickDoneAssigneeButton();
+	const user2 =
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'test@liferay.com'
+		);
 
-	const user2Id = parseInt(user2.id);
+	await workflowReviewTasksPage.selectAssignee(user2.id.toString());
 
-	await apiHelpers.headlessAdminUser.deleteUserAccount(user2Id);
+	await workflowReviewTasksPage.clickDoneAssigneeButton();
```

Comment:
Hey, Victor. Can we add an additional step to the test by asserting that the user was directed to the desired page?